lib/utils.js

```diff
@@ -39,11 +39,9 @@ function normalizeBinary (binaryPath, platform, arch) {
       var channelNames = ["firefox", "firefoxdeveloperedition", "beta", "nightly", "aurora"];
 
       if (channelNames.indexOf(binaryPath) !== -1) {
-        // Try to find the app for this channel.
-        // mdfind "kMDItemCFBundleIdentifier == 'org.mozilla.firefoxdeveloperedition'"
-        var results = spawnSync("mdfind", ["kMDItemCFBundleIdentifier == 'org.mozilla." + binaryPath + "'"]);
-        if (results.stdout) {
-          result = results.stdout.toString().split('\n')[0];
+        var macApp = findMacAppByChannel(binaryPath);
+        if (macApp) {
+          result = macApp;
```

Comment:
I think we could just assign the result straight from the call, since result will be `null` before we get here, and we'll either assign it to something or `null`…